ext/pgsql/pgsql.c

```diff
@@ -2833,6 +2833,36 @@ PHP_FUNCTION(pg_set_error_verbosity)
 }
 /* }}} */
 
+PHP_FUNCTION(pg_set_error_context_visibility)
+{
+	zval *pgsql_link = NULL;
+	zend_long visibility;
+	PGconn *pgsql;
+	pgsql_link_handle *link;
+
+	if (ZEND_NUM_ARGS() == 1) {
+		if (zend_parse_parameters(ZEND_NUM_ARGS(), "l", &visibility) == FAILURE) {
+			RETURN_THROWS();
+		}
+		link = FETCH_DEFAULT_LINK();
+		CHECK_DEFAULT_LINK(link);
+	} else {
```

Comment:
Considering we have deprecated the fetching of the default link I would rather not introduce a new function with this functionality. Would also make the stubs correct.

Reply:
oky doky